wsl-pro-service/internal/system/system.go

```diff
@@ -274,3 +278,33 @@ func (s *System) findCmdExe() (cmdExe string, err error) {
 
 	return "", fmt.Errorf("none of the mounted drives contains subpath %s", subPath)
 }
+
+// lookupGroup searches the group with the specified name and returns its GID.
+func (s *System) lookupGroup(name string) (int, error) {
```

Comment:
I would rename to something more explicit, like groupToGID as it doesn’t have the same signature than the rest of the lookup functions in go